templates/patterns/list_item/pattern-list-item--variant-thumbnail-primary.html.twig

```diff
@@ -4,12 +4,17 @@
  * Default theme implementation for a ECL thumbnail list item component.
  */
 #}
-{% include '@ecl/ec-component-list-item' with {
-  'variant': 'thumbnail',
-  'title': title,
-  'metas': meta,
-  'href': url,
-  'detail': detail,
-  'primary_image': image,
-  'screen_reader': 'List item'|t
+{% set _image_position = image_position|default('before') %}
```

Comment:
No need to set a variable for this one, let's directly assign it to the value below:
```
     image_position: image_position|default('before'),
```
It will increase Twig file readability.

Reply:
True.